### ISSUE TITLE
Issue with modal

### ISSUE THREAD

**Issue by MihailZharavin11** [mskocik/svelty-picker#112]
Hi! I have issue with modal. In other modal windows everything works, in this one I encountered a problem. When opening a modal window, I get an error: Cannot read properties of null (reading 'x'). The picker automatically opens and gives this error.
![image](https://github.com/mskocik/svelty-picker/assets/43700069/d67cef0a-a56b-4070-98d5-7314b5820f0e)

**Comment by mskocik** [comment]
I would need more details, provided screen is not enough

**Comment by MihailZharavin11** [comment]
Contents of the modal window
![image](https://github.com/mskocik/svelty-picker/assets/43700069/aefe12c9-6dd1-4c7e-b07a-a8a4c2ff3548)

**Comment by mskocik** [comment]
Also if you using svelty-picker v3, try v4 instead

**Comment by MihailZharavin11** [comment]
You're right, it's because of autofocus, if you place another input or something else on top of the element, the problem goes away, also you can change tabindex. Thank you !

**Comment by MihailZharavin11** [comment]
You're right, it's because of autofocus, if you place another input or something else on top of the element, the problem goes away, also you can change tabindex. Thank you !

**Comment by mskocik** [comment]
Great to hear!
